app/script/calling/entities/FlowEntity.js

```diff
@@ -862,7 +869,9 @@ z.calling.entities.FlowEntity = class FlowEntity {
 
         const isModeDefault = this.negotiationMode() === z.calling.enum.SDP_NEGOTIATION_MODE.DEFAULT;
         if (isModeDefault && sendingOnTimeout) {
-          const connectionConfig = this.peerConnection.getConfiguration();
+          const connectionConfig = this.peerConnection.getConfiguration
+            ? this.peerConnection.getConfiguration()
+            : this.peerConnectionConfiguration;
```

Comment:
I would propose the following in case anything else is wrong in the implementation:

`const connectionConfig = this.peerConnection.getConfiguration && this.peerConnection.getConfiguration() || this.peerConnectionConfiguration;`